src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -146,7 +146,7 @@ def ignore_stack_frame(self, stack_frame):
       return True
 
     return False
-  
+
```

Comment:
Is this change adding a tab or something?

Reply:
That's strange Jonathan, I don't see anything in the "Files Changed" diff for lines 149 or 3598.

Reply:
Ha must've been an old commit.